fixing profilePage

ishimweeli opened the pull request:
# PR Description
displaying info from database on profile page
Please include a brief description of the pull request you want to raise.

# Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you or that you assigned yourself. Make sure you check evey completed task.

# How has this been tested?
after logging in as a user click on profile you will be able to see you manager organisation you belong to phase you are in and program
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.

# Number of Commits
1
The number of commits should not exceed **2 commits**. In case they are more than that, please take your time and squash them.

# Screenshots (If appropriate)

# Please check this Checklist before you submit your PR:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My code generate no warnings
- [ ] My test coverage meet the set test coverage threshold
- [ ] There are no vulnerabilities
- [ ] There are no conflicts with the base branch

ceelogre reviewed (commented):
The role attribute for admin is different on about and organization tabs.
<img width="346" alt="image" src="https://user-images.githubusercontent.com/5022823/208391644-d7f945dc-25eb-4398-8c2d-90d7e3617890.png">
Also, what is the meaning of Unavailable?

<img width="346" alt="image" src="https://user-images.githubusercontent.com/5022823/208391700-051e10f0-bdc7-43cf-bd98-8606aad30777.png">

alaintresor committed:
fix(role):fix roles (#198)

-fix roles issue

[work in process]

fixing profilePage

profileupdated

profileupdated and fixed

profileupdated and fixed real

profileupdated and fixed realedit

profileupdated and fixed realedit

fixing edit page

fixing edit page

fizing organisation on profile page

fizing organisation on profile page

fixing organisation on profile page

fixing organisation on profile page

fix-profilepage-new-commit

profilePage

ceelogre reviewed (dismissed):
There's a bad request error.
<img width="1430" alt="image" src="https://user-images.githubusercontent.com/5022823/208932540-d257615c-cea0-4555-9395-70bc76dbc911.png">
